authz/authz.proto

```diff
@@ -28,25 +28,50 @@ option go_package = "github.com/openconfig/gnsi/authz";
 //
 
 // Authorization Policy defines which principals are permitted to access which
-// resource. Resources are individual RPC methods scoped by services.
+// resource(s). Resources are individual RPC methods scoped by their path(s).
+// The policy is expressed in JSON format, following the schema in
+// https://github.com/grpc/proposal/blob/master/A43-grpc-authorization-api.md
+// and the protobuf in the
+// README(https://github.com/haussli/gnsi/blob/main/authz/README.md).
```

Comment:
Did you mean https://github.com/openconfig/gnsi/blob/main/authz/README.md ?

Reply:
Oops, yes.   Thanks.